include/editor.hpp

```diff
@@ -0,0 +1,200 @@
+#pragma once
```

Comment:
Any specific reason why we need editor class in P11?

In P10, we had two copies of vpd parse logic - one in impl.cpp and another in editor.cpp. since we can't change the working code, we left it untouched.

Now in P11, design has been changed and we have separate classes for each parser type and parse logic is already implemented in respective classes.

Here editor class again has the parse logic plus modify vpd logic. why can't we get rid of editor class completely and have modify vpd logic clubbed to the respective classes (say in ipz_parser.cpp or kw_parser.cpp).

Reply:
It has been discussed before and that's the plan.
Instead of editor class each parser will have its own write API.

That is why editor has not been merged. We will not take it further. 

Please stop reviewing this commit further. We will just keep it here till we implement write under IPZ so that any code can be re-used if required. Post which the PR can be abandoned.